React/src/pages/warehouses/helper.ts

```diff
@@ -32,7 +32,7 @@ export const headCells: Array<HeadCell<WarehouseFilterDto>> = [
         type: 'string',
         id: 'fullAddress',
         label: 'Adres',
-        visibleInMode: [AppSize.mobile, AppSize.tablet, AppSize.notebook, AppSize.desktop],
+        visibleInMode: [AppSize.tablet, AppSize.notebook, AppSize.desktop],
```

Comment:
To też jest dosyć ważna informacja, która moim zdaniem powinna być wyświetlana bez przejścia w szczegóły.